baseplate/events/queue.py

```diff
@@ -65,6 +75,9 @@ def set_field(self, key, value, obfuscate=False):
         :param bool obfuscate: Whether or not to put the field in the obfuscated
             section. This is used for sensitive info like IP addresses that must
             be treated with care.
+        :param bool high_cardinality: Whether or not to put the field in the
+            interana_excluded section.  This excludes the field from being
```

Comment:
I'm kinda torn here. I really do prefer the high cardinality (or similar) naming rather than explicitly calling out interana, since that's very leaky of current tooling. But I also understand that we want to be obvious to users that this is how they avoid putting stuff into interana since that's what they're using. Maybe the high cardinality option should be described as "for fields that should not be indexed due to high cardinality (e.g. not used in Interana)"?